AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -222,18 +214,56 @@ private extension SessionCartView {
                         .frame(height: 120)
                         .disabled(true)
                     HStack() {
-                            Text(startTime)
+                            startTime
                             Spacer()
                         Text("\(Strings.SessionCartView.avgSession) \(selectedStream.unitSymbol ?? "")")
                             Spacer()
-                            Text(endTime)
+                            endTime
                     }.foregroundColor(.aircastingGray)
                     .font(Font.muli(size: 13, weight: .semibold))
                 }
             }
         }
     }
     
+    var startTime: some View {
+        let formatter = Constants.dataFormatter
+        
+        if !(session.isMobile && session.isActive && session.deviceType == .MIC) {
+            formatter.timeZone = TimeZone.init(abbreviation: "UTC")
+        }
+            
+            guard var start = session.startTime else { return Text("") }
+     
+        if session.isFixed && session.measurementStreams == [] {
```

Comment:
`session.measurementStreams == []` do you mean that it's empty? Maybe `isEmpty` would be more descriptive? ;D

Reply:
as it is @NSMenaged is cannot be done with .isEmpty :/